scalameta/semanticdb-scalac-core/src/main/scala/scala/meta/internal/semanticdb/DocumentOps.scala

```diff
@@ -253,6 +253,12 @@ trait DocumentOps { self: DatabaseOps =>
               val mname = mctorrefs(gpoint)
               gtree match {
                 case g.Select(_, g.nme.CONSTRUCTOR) => success(mname, gtree.symbol)
+                case gtree: g.ClassDef =>
```

Comment:
we'll need to guard against `config.members.isAll` since this behavior is opt-in